conftest.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import shutil
+
+import pytest
+
+from utilities.pytest_utils import (
+    separator,
+)
+
+LOGGER = logging.getLogger(__name__)
+BASIC_LOGGER = logging.getLogger("basic")
+
+
+def pytest_report_teststatus(report, config):  # type: ignore
```

Comment:
Why type ignore in these functions?

Reply:
because these are pytest implementations, you cannot call them. they have typing in pytest code. no need to repeat here